src/cpp/src/generation_config.cpp

```diff
@@ -24,6 +24,10 @@ GenerationConfig::GenerationConfig(const std::string& json_path) {
     read_json_param(data, "max_new_tokens", max_new_tokens);
     read_json_param(data, "max_length", max_length);
     // note that ignore_eos is not present in HF GenerationConfig
+    read_json_param(data, "min_new_tokens", min_new_tokens);
+    read_json_param(data, "stop_strings", stop_strings);
+    // note that include_stop_str_in_output is not present in HF GenerationConfig
+    // note that stop_token_ids is not present in HF GenerationConfig
```

Comment:
You can still try reading `include_stop_str_in_output` and `stop_token_ids`. Maybe someone puts them into the config for you. Although the comment should not be removed explaining that it's not common to have them in the config.

I had the same comment about `ignore_eos`. Apparently it was ignored back then.